cookiecutter-django-ida/{{cookiecutter.repo_name}}/{{cookiecutter.project_name}}/urls.py

```diff
@@ -19,7 +19,7 @@
 
 from auth_backends.urls import oauth2_urlpatterns
 from django.conf import settings
-from django.conf.urls import include, url
+from django.urls import re_path,include
 from django.contrib import admin
```

Comment:
```suggestion
from django.contrib import admin
from django.urls import include, re_path
```
Something's wrong in the quality checks -- the isort call here should have detected this: https://github.com/openedx/edx-cookiecutters/blob/f49b8beb757ae250747dea191e57e6ce037d9686/tests/test_cookiecutter_django_ida.py#L116

Reply:
@timmc-edx sorry for late reply, I got distracted and just now by coincidence I stumbled upon this PR again